src/core/cpuid.d

```diff
@@ -497,7 +509,9 @@ void getcacheinfoCPUID2()
     // for old single-core CPUs.
     uint numinfos = 1;
     do {
-        asm pure nothrow @nogc {
+        version(GNU) asm {
```

Comment:
For whatever reason I believed this syntax doesn't work. I'll add them everywhere.